easydock/database.py

```diff
@@ -305,15 +324,16 @@ def add_protonation(db_fname, tautomerize=True, table_name='mols', add_sql=''):
             fd, output = tempfile.mkstemp()  # use output file to avoid overflow of stdout in extreme cases
             try:
                 for smi, _, mol_id in data_list:
-                    tmp.write(f'{smi}\t{mol_id}\n')
+                    tmp.write(f'{smi}\t{mol_id}\t{stereo_id}\n')
```

Comment:
I think we have to join mol_id and stereo_id, otherwise stereo_id may not become a part of a title of an output molecule
```python
tmp.write(f'{smi}\t{mol_id}_{stereo_id}\n')
```